chaincode-labs/chaincode-podcast/the-bitcoin-development-kit-bdk.md

```diff
@@ -328,39 +319,45 @@ So we kind of explored the ideas for...
 So at one point I wanted to make BDK generic over the amount type.
 So the idea was to use Rust generic so that you can have...
 You can use any type for your amount as long as you can basically sum two amounts so you can compare them so I can say this is greater than that.
-So the idea was if I, for Bitcoin you would use an unsigned 64-bit integer but for liquid you would use their, whatever, Pedersen commitment type, because you can sum them together, but you can't really compare them, so you can't say is this greater than this one because it's binary, so I tried to implement that and it was a mess, so I stopped.
+So the idea was if I, for Bitcoin you would use an unsigned 64 bit integer but for liquid you would use their, whatever, Pedersen commitment type, because you can sum them together, but you can't really compare them, so you can't say is this greater than this one because it's binary, so I tried to implement that and it was a mess, so I stopped.
 So yeah, I think that that's one of the roadblocks, otherwise they would probably be happy to use it, I think.
+
+Adam Jonas & Mark Erhardt: 00:16:21
```

Comment:
I appreciate this, but we don't really support this kind of "double" speaker attribution. I removed one speaker to stick with the supported format.